src/parquet/column_reader.cc

```diff
@@ -167,8 +164,10 @@ bool TypedColumnReader<TYPE>::ReadNewPage() {
   return true;
 }
 
-std::shared_ptr<ColumnReader> ColumnReader::Make(const parquet::ColumnMetaData* metadata,
-    const parquet::SchemaElement* element, InputStream* stream) {
+std::shared_ptr<ColumnReader> ColumnReader::Make(
+                                const parquet::ColumnMetaData* metadata,
+                                const parquet::SchemaElement* element,
+                                std::unique_ptr<InputStream>& stream) {
```

Comment:
This indentation (similarly below) is inconsistent with the rest of the codebase. Since we are sharing code with Impala (and will continue to) and Kudu, we should indent subsequent lines of function signatures 4 spaces.